xcdat/spatial.py

```diff
@@ -178,14 +178,98 @@ def average(
                 self._validate_region_bounds("Y", lat_bounds)
             if lon_bounds is not None:
                 self._validate_region_bounds("X", lon_bounds)
-            dv_weights = self._get_weights(axis, lat_bounds, lon_bounds)
+            dv_weights = self.get_weights(axis, lat_bounds, lon_bounds)
         elif isinstance(weights, xr.DataArray):
             dv_weights = weights
 
         self._validate_weights(dv, axis, dv_weights)
         dataset[dv.name] = self._averager(dv, axis, dv_weights)
         return dataset
 
+    def get_weights(
```

Comment:
I assume this is the only line that changed in this file? Github is highlighting this whole function...probably because it got moved?

Reply:
Yup, the method was renamed, moved, and its references were updated. Nothing else was changed here.

Reply:
Yes, this method was moved and renamed. Nothing else was changed in this file.